src/dndbeyond/base/utils.js

```diff
@@ -167,6 +167,8 @@ function buildAttackRoll(character, attack_source, name, description, properties
                     }
                 }
             }
+            if (roll_properties.name === "Blade of Disaster")
+                crit_damages[0] = crit_damages[0] = damagesToCrits(character, ["8d12"])[0];
```

Comment:
you have `crit_damages[0] = ` appearing twice in the line.

Reply:
That's funny, it worked though 😂

Reply:
Just trying to give the JS engine some work in these poor economic times, obviously.